packages/amplication-code-gen-types/src/code-gen-types.ts

```diff
@@ -148,3 +149,20 @@ export type ResourceGenerationConfig = {
   dataServiceGeneratorVersion: string;
   appInfo: AppInfo;
 };
+
+export enum EnumMessagePatternConnectionOptions {
+  "None" = "None",
+  "Receive" = "Receive",
+  "Send" = "Send",
+}
```

Comment:
The dsg types should be based on the server types and we should not recreated them from scratch

Reply:
Because of that we removed the old enum from the server